functions/src/common/cleanComplexTable.ts

```diff
@@ -1,11 +1,35 @@
-// has 2 main columns, each also has 2 colums, 1 - spec, 2 - value
-// has categories, like: Main, General, GPU, RAM...
-// https://ek.ua/en/ASUS-GEFORCE-RTX-3060-DUAL-V2-OC-LHR.htm
-const cleanComplexTable = (rawSpecsTable: any) => {
-  const cleanedSpecsTable = rawSpecsTable
-    .split('\n')
-    .filter((item: string) => item.includes('\t'));
-  return cleanedSpecsTable;
-};
-
-export default cleanComplexTable;
+// has 2 main columns, each also has 2 colums, 1 - spec, 2 - value
+// has categories, like: Main, General, GPU, RAM...
+// https://ek.ua/en/ASUS-GEFORCE-RTX-3060-DUAL-V2-OC-LHR.htm
+const cleanComplexTable = (rawSpecsTable: any) => {
+  const cleanedSpecsTable = rawSpecsTable.split('\n');
+
+  const normalizedSpecsTable = cleanedSpecsTable.filter((item: string) => {
+    if (item.includes('Socket')) {
+      const indexOfSocketSpec = cleanedSpecsTable.indexOf(item);
+      const lastIndexOfSocketSpec = cleanedSpecsTable.indexOf(
+        '',
+        indexOfSocketSpec,
+      );
```

Comment:
why you simply don`t do indexOfSocketSpec + 1?

Reply:
because i need index where sockets end, if i use indexOfSocketSpec + 1 there will be 2 sockets